cosmo_tester/test_suites/broker_management/manager_test.py

```diff
@@ -0,0 +1,79 @@
+import json
+
+
+def test_broker_management(brokers_and_manager, logger):
+    # All in one test for speed until such time as complexity of these
+    # operations increases to the point that extra tests are needed.
+    broker1, broker2, manager = brokers_and_manager
+    manager.put_remote_file(
+        local_path=broker1.ca_path,
+        remote_path='/tmp/rabbit.ca',
+    )
+
+    manager.additional_install_config = {
+        'rabbitmq': {
+            'ca_path': '/tmp/rabbit.ca',
+            'cluster_members': {
+                broker1.hostname: {
+                    'default': str(broker1.private_ip_address),
```

Comment:
it's weird that we'd need those str() everywhere around the private_ip_address, i don't think we need them

Reply:
Apparently config.yaml gets upset when the IP address is !!python-unicode 192.0.2.56